frontend/test/services/contract.test.js

```diff
@@ -20,51 +22,6 @@ describe('Contract Deployment Tests', () => {
     });
   });
 
-  describe('deployContract', () => {
```

Comment:
you must not deleted tests. it should be the same amount as we have in main now, I see 8 less

Reply:
unresolved